xinference/core/chat_interface.py

```diff
@@ -173,6 +181,130 @@ def generate_wrapper(
             analytics_enabled=False,
         )
 
+    def build_chat_vl_interface(
+        self,
+    ) -> "gr.Blocks":
+        def predict(history, bot):
+            logger.debug("Predict.")
```

Comment:
Is this a duplicated logger when you test?

Reply:
Fixed.